nautobot/ipam/tests/test_filters.py

```diff
@@ -426,15 +426,15 @@ class IPAddressTestCase(FilterTestCases.FilterTestCase, FilterTestCases.TenancyF
     def setUpTestData(cls):
         # Create some VRFs that belong to the same Namespace and have an rd
         cls.namespace = Namespace.objects.create(name="IP Address Test Case Namespace")
-        VRF.objects.create(name="VRF 1", rd="65000:100", namespace=cls.namespace)
-        VRF.objects.create(name="VRF 2", rd="65000:200", namespace=cls.namespace)
-        VRF.objects.create(name="VRF 3", rd="65000:300", namespace=cls.namespace)
+        vrfs = (
+            VRF.objects.create(name="VRF 1", rd="65000:100", namespace=cls.namespace),
+            VRF.objects.create(name="VRF 2", rd="65000:200", namespace=cls.namespace),
+            VRF.objects.create(name="VRF 3", rd="65000:300", namespace=cls.namespace),
+        )
         # Create some VRFs without an rd
         VRF.objects.create(name="VRF 4", namespace=cls.namespace)
         VRF.objects.create(name="VRF 5", namespace=cls.namespace)
         VRF.objects.create(name="VRF 6", namespace=cls.namespace)
-        vrfs = VRF.objects.filter(namespace=cls.namespace, rd__isnull=False)
-        assert len(vrfs) == 3, f"This Namespace {cls.namespace} does not contain enough VRFs."
```

Comment:
Looks like we're going to enforce not using the `assert` keyword due to https://docs.astral.sh/ruff/rules/assert/#assert-s101 ?

Reply:
Right.